### PR TITLE
update instructions for registering a squid (SOFTWARE-3310)

### PR DESCRIPTION
**Pull request by edquist**

### CODE REVIEW

File: docs/data/frontier-squid.md [hunk 1]
```diff
@@ -143,6 +142,22 @@ If your squid will be supporting the Frontier application, it is also
 good to do the test in the
 [upstream documentation Testing the installation section](https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Testing_the_installation).
 
+## Registering Frontier Squid
+
+See the [registration doc](/common/registration#new-registrations) for instructions on registering a new resource.
+(Or see the instructions for [updating existing registrations](/common/registration#updating-existing-registrations)
+if you are adding the squid service to an existing resource or resource group.)
+You will be using `template-resourcegroup.yaml` for your reference template to register various information about your
```
Comment (brianhlin) [inline docs/data/frontier-squid.md]: Let's link directly to the `template-resourcegroup.yaml` in the topology repository

File: docs/data/frontier-squid.md [hunk 2]
```diff
@@ -143,6 +142,22 @@ If your squid will be supporting the Frontier application, it is also
...
+resource.
+
+Additionally:
+
+- You will want to add a `Squid:` section to the `Services:` list, with any relevant fields for that service.
```
Comment (brianhlin) [inline docs/data/frontier-squid.md]: Drop `you will want to`, it's in our list of superfluous statements https://opensciencegrid.org/technology/documentation/writing-documentation/#tips-for-writing-procedural-instructions

File: docs/data/frontier-squid.md [hunk 3]
```diff
@@ -143,6 +142,22 @@ If your squid will be supporting the Frontier application, it is also
...
+
+- If you are setting up a new resource, you will want to set `Active:` to `false`.
```
Comment (brianhlin) [inline docs/data/frontier-squid.md]: > If you are setting up a new OSG site, set `Active: false`.

File: docs/data/frontier-squid.md (hunk 1 above)
Comment (brianhlin) [reply docs/data/frontier-squid.md]: Also, `Use template-resourcegroup.yaml...` instead of `You will be using`

File: docs/data/frontier-squid.md [hunk 4]
```diff
@@ -143,6 +142,22 @@ If your squid will be supporting the Frontier application, it is also
...
+  The `Active:` field for a resource should only be set to `true` when it is accepting requests and ready for
+  production.
```
Comment (brianhlin) [inline docs/data/frontier-squid.md]: > Only set `Active: true` when your squid resource is ready for production.

**Comment by edquist** [comment]
Arright @brianhlin, i think i've addressed your comments.

File: docs/data/frontier-squid.md [hunk 5]
```diff
@@ -142,6 +142,19 @@ If your squid will be supporting the Frontier application, it is also
...
 
+## Registering Frontier Squid
+
+See the [registration doc](/common/registration#new-registrations) for instructions on registering a new resource.
+(Or see the instructions for [updating existing registrations](/common/registration#updating-existing-registrations)
+if you are adding the squid service to an existing resource or resource group.)
+
+Additionally:
+
+1.  Add a `Squid:` section to the `Services:` list, with any relevant fields for that service.
```
[the remark this answers is not in the document]
Comment (edquist) [reply docs/data/frontier-squid.md]: I even provided some descriptive text!

**Pull request merged**